package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "justgage",
-  "version": "1.5.0",
+  "version": "1.5.1",
```

Comment:
Don't bump version, the release method automatically handles this

Reply:
Reverted version# bump.